### DISCUSSION TITLE
Add previous plays at the end screen for graveyard maps

### DISCUSSION THREAD

**Discussion by KorigamiK** [ppy/osu#25476] [Ideas]
It's bad as it is, having no learderboard for graveyard maps. If I could just compare my performance with my previous plays, I think it would be a huge win for everyone.

**Comment by Theighlin** [comment]
Do personal previous plays not show if you switch the leaderboard to show "local" scores in song select?

**Reply by peppy** [reply]
I think they should show without being required to switch things at song select. I really don't think the results screen should be tied to the song select leaderboard in any way.

**Reply by Theighlin** [reply]
Yeah, I'd separate the two. Show local scores by default on maps with no leaderboards, and add the same song select switch at results (one might be interested in other leaderboards temporarily after a map but not while searching for one)
